### ISSUE TITLE
yelpapi.yelpapi.YelpAPIError: TOKEN_MISSING:

### ISSUE THREAD

**Issue by pmdbt** [gfairchild/yelpapi#9]
Hi, this is my first time posting on GitHub, so please bare with me. It is my understanding that in v3 of Yelp's Fusion API, the access token was deprecated, so authentication only requires an API_KEY now. 

When I tried using the demo code provided: 
 from yelpapi import YelpAPI
yelp_api = YelpAPI(api_key)
search_results = yelp_api.search_query(args)

I obtain an error telling me that I have a missing token, which is confusing, since yelp claims we no longer need that field. 

Thank you for any input you can provide on this issue.

**Comment by pmdbt** [comment]
Also, I wanted to add that of course I have declared the variable api_key and stored my actual api key in it before I ran the code.

**Comment by gfairchild** [comment]
Can you verify the version of yelpapi you currently have? Also, please copy/paste the *exact* error you're getting (not just the error message but the full output from when you hit enter to run your script).

**Comment by gfairchild** [comment]
Cool, I've got some ideas, but I need to do some testing. I'm currently on vacation, so I'll respond by Tuesday.

**Comment by pmdbt** [comment]
I've actually been able to fix it, so no need to stress out on your end. It turns out that other people have  this issue too using Yelp's v3 Fusion api off of their own GitHub. It seems that if a dev had an API_KEY from before the TOKEN was deprecated, the system still assumes that a TOKEN is needed for access to Yelp. I was able to fix the issue by simply updating and generating a new API_KEY. I tested that with Yelp's example python code and it worked. I then tested it with your yelpapi and it worked as well. Thanks for everything, and I'm not sure how to close an issue raised on GitHub. But, feel free to close this thread as the issue has been resolved.
